io-engine/src/bin/io-engine-client/bdev_cli.rs

```diff
@@ -134,13 +137,15 @@ async fn create(mut ctx: Context, args: &ArgMatches<'_>) -> crate::Result<()> {
         })?
         .to_owned();
 
-    let response = ctx
-        .bdev
-        .create(BdevUri {
-            uri,
-        })
-        .await
-        .context(GrpcStatus)?;
+    // let response = ctx
```

Comment:
If it's mot needed, can we just delete these lines ?

Reply:
I am yet to test all changes . Hence kept older calls. WIll test it tomorrow and remove.

Reply:
We should keep v0 support somehow, maybe as a different module, at least until we remove v0 api completely.

Reply:
Incorporated